src/status_im2/contexts/shell/jump_to/components/floating_screens/style.cljs

```diff
@@ -13,9 +15,16 @@
     :z-index       screen-z-index}
    {:background-color (colors/theme-colors colors/white colors/neutral-95)
     :overflow         :hidden
-    :position         :absolute}))
+    ;; KeyboardAvoidingView which is used for chat screen composer,
+    ;; not working when we use :absolute layout. One fix is to add
+    ;; KeyboardAvoidingView :behaviour height in android, which is also
+    ;; recommended in the documentation. It fixes KeyboardAvoidingView but
+    ;; the pushing of views by the keyboard is not smooth & while animating it creates a weird jump.
```

Comment:
Kind of related, adding for context
https://github.com/facebook/react-native/issues/14825
https://stackoverflow.com/questions/44909375/react-native-keyboardavoidingview-not-animated

Reply:
Can you create an issue for this and add in a TODO with the issue number??

Reply:
Hi @J-Son89, thank you very much for reviewing PR.
Not sure what I should report in the issue as there is no bug or anything that needs to be fixed. I just added a comment to explain the reasoning behind why I used a relative layout here and why we should keep it.

Reply:
oh okay, I think it might be the phrasing that confused me here a bit.
i.e "One fix is to add ..."

Perhaps a slight rephrasing is less confusing (or perhaps it's just me):
    ;; KeyboardAvoidingView which is used for chat screen composer,
    ;; not working when we use :absolute layout. Using :relative instead, although an alternative solution is to add
    ;; KeyboardAvoidingView :behaviour height in android, which is also
    ;; recommended in the documentation. This fixes KeyboardAvoidingView but
    ;; the pushing of views by the keyboard is not smooth & while animating it creates a weird jump.

Reply:
Ideally there is a way to restructure the composer so we can keep it `:absolute` all of the time. imo it's not so nice to have this based on specific view-id's